c7n/manager.py

```diff
@@ -68,4 +72,8 @@ def filter_resources(self, resources, event=None):
     def get_model(self):
         """Returns the resource meta-model.
         """
-        return self.query.resolve(self.resource_type)
+        from exceptions import AttributeError
+        try:
+            return self.query.resolve(self.resource_type)
+        except AttributeError as e:
+            raise ValidationError(e.message)
```

Comment:
per in person discussion this isn't an appropriate place to raise a validation error, that should be an explicit path closer to the user, vs this implicit path in the base class of resource manager, as it will mask actual issues.